Libraries/libutil/Sources/DefaultProcessContext.cpp

```diff
@@ -112,6 +112,29 @@ executablePath() const
 #endif
 }
 
+static int commandLineArgumentCount = 0;
```

Comment:
Rename these to make it clear they're static arguments, like sCommandLineArgumentCount?